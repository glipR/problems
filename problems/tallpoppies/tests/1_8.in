4 72 69
670024051 711637357 607436006 439167236 6312456 557996606 866342998 83815424 498649004 769914035 88252842 743305336 28172148 679160145 776089742 942468827 199431222 123501440 455208863 516225827 968475648 838318483 565511578 172966636 213791511 387042409 65255364 460113968 188516823 808298500 717316578 810374915 643317812 66945529 237083746 639505767 257032243 654173891 384696701 584420635 879165673 320586020 826886451 507637471 964582868 817446614 762688994 222196649 391708223 194483165 562691853 80500535 110848644 382419148 945548779 747031856 661152248 770199494 306550727 891808475 67611491 243884876 300909315 751434615 290892624 53457257 606134827 983430614 299940096 935125982 113299211 378872501
2493603 0
7912783 1 724634219
9873401 1 188122268
25899495 1 23358918
32833849 1 403458925
37750553 0
67238116 1 924812160
69773518 1 857504047
85967688 0
99001613 0
123287526 1 520946177
123516686 0
128663915 0
153800359 0
165797926 1 828750830
177715575 1 758034179
189825883 1 955700965
201504578 0
215957753 0
216685228 0
231448376 0
244933500 0
262851318 0
277670800 1 76384709
284874008 0
308046820 1 186699652
344026035 0
345772204 0
364498325 0
398715533 0
400154001 0
401392086 0
434954264 0
448167740 1 926361536
449308560 1 521513
463750632 0
464259716 0
464560446 0
465841479 1 388845148
472448641 0
477128698 1 509206061
529293245 0
534931221 1 501340553
570592378 1 538160282
595840239 0
634357691 0
639719650 1 561878469
666637232 0
676188267 1 240677749
676793085 1 611257736
686267092 1 550237173
691867325 1 332758741
706203494 1 724689429
765352631 0
790309187 1 898179738
807500392 0
814139211 0
820060708 1 399578506
826217123 0
830433865 1 346191610
837633200 1 535773412
850148478 1 664672517
856359659 1 769856420
868971936 0
945729184 0
970582573 0
971451517 1 73579676
972435343 1 554880769
994687739 0
