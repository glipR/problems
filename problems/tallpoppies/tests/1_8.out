993405026
1119477860
1308453620
1341555231
1436535571
1449781642
1507009911
1685183985
1730174010
1705059395
1752679955
1797180614
1861780187
1947794532
2152193882
2153288310
2227907335
2357551126
2352050619
2352600200
2483122392
2572319106
2568676221
2551224871
2568954709
2787197031
3044513204
3193290460
3320722819
3704728336
3869507335
3863992850
3911003319
4060308379
4361326711
4447841870
4541442809
